dts/bindings/sensor/epcos,b57861s0103a039.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+description: EPCOS, B57861S0103A039 NTC Thermistor
+
+compatible: "epcos,b57861s0103a039-spi"
```

Comment:
Why is spi needed in the compatible name? The thermistor could be connected to any adc presumably, including the builtin soc adc?